### ISSUE TITLE
Choose A Topic Screen

### ISSUE THREAD

**Issue by baljindersidhu** [baljindersidhu/Quizmania#1]
The next button on screen is not taking to next activity neither is the app crashing.

**Comment by deepannegi** [comment]
I don't think that you pushed the new code. GitHub still has the old one with the splash screen.

Sent from Windows Mail

From: Baljinder Singh
Sent: ‎Sunday‎, ‎October‎ ‎18‎, ‎2015 ‎3‎:‎59‎ ‎AM
To: baljindersidhu/Quizmania

The next button on screen is not taking to next activity neither is the app crashing.

**Comment by baljindersidhu** [comment]
Did you check the new app I sent you in fb?
